science/tetgen/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+
+name                    tetgen
+version                 1.6.0
+revision                1
```

Comment:
For new ports, `revision` should start at 0.

`revision` is sort of like a sub-version that can be updated when a port is changed but its version number is staying the same. `revision` will usually reset to 0 on every `version` bump.